colcon_bundle/verb/utilities.py

```diff
@@ -9,10 +9,39 @@
 import subprocess
 import sys
 
-
 from colcon_bundle.verb import logger
 
 
+def get_ros_distribution_version():
+    """
+    Discover and return ROS distribution version.
+
+    :return: the ROS distribution version to be used.
+    example: kinetic, melodic
+    """
+    ros_distribution_version = 'kinetic'
+    if get_ubuntu_distribution_version() == 'bionic':
+        ros_distribution_version = 'melodic'
+    return ros_distribution_version
+
+
+def get_ubuntu_distribution_version():
+    """
+    Discover and return Ubuntu distribution version.
+
+    :return: the Ubuntu distribution version of the build server.
+    example: xenial, bionic
+    """
+    import distro
+    distribution = distro.linux_distribution()
+    if distribution[0] == 'Ubuntu' and distribution[1] == '16.04':
+        return 'xenial'
+    elif distribution[0] == 'Ubuntu' and distribution[1] == '18.04':
+        return 'bionic'
+    else:
+        raise ValueError('Unsupported distribution', distribution)
```

Comment:
It's bad practice to prevent users from using code wherever they would like to.
Are we 100% sure it won't work? If not, let's warn the user (we may also tell the user to pass an additional flag `--i-know-what-imdoing` to confirm but we should not block people from running the script.

If it actually works (even just by chance), it is infuriating for users to be blocked because of that type of check.